packages/icons/src/extraIconSets/misc.ts

```diff
@@ -0,0 +1,680 @@
+      'ReplyMirrored': '\uEE35',
+      'ReportLibrary': '\uEEBB',
+      'ReportLibraryMirrored': '\uEEBC',
+      'ReturnToSession': '\uED24',
+      'RevToggleKey': '\uE845',
+      'Ribbon': '\uE9D1',
+      'RightDoubleQuote': '\uE9B1',
+      'Ringer': '\uEA8F',
+      'Room': '\uED9F',
+      'Rotate': '\uE7AD',
+      'Rugby': '\uEB2D',
+      'Running': '\uEADA',
+      'Sad': '\uE757',
+      'Save': '\uE74E',
+      'SaveAs': '\uE792',
+      'Section': '\uEC0C',
+      'Sections': '\uEF76',
+      'SecurityGroup': '\uED85',
+      'Send': '\uE724',
+      'SendMirrored': '\uEA63',
+      'SetAction': '\uF071',
+      'ShareiOs': '\uEF79',
+      'SharepointLogo': '\uED18',
+      'Shield': '\uEA18',
+      'Shop': '\uE719',
+      'ShoppingCart': '\uE7BF',
+      'ShowResults': '\uE8BC',
+      'ShowResultsMirrored': '\uEA65',
+      'SidePanel': '\uEF52',
+      'SingleBookmark': '\uEDFF',
+      'SipMove': '\uE759',
+      'SkypeCircleCheck': '\uEF7D',
+      'SkypeCircleClock': '\uEF7E',
+      'SkypeCircleMinus': '\uEF7F',
+      'SkypeLogo': '\uEB6F',
+      'SkypeMessage': '\uEF83',
+      'SliderThumb': '\uEC13',
+      'Snow': '\uEB46',
+      'SnowShowerDay': '\uE9FD',
+      'SnowShowerNight': '\uEA11',
+      'Soccer': '\uEB21',
+      'SocialListeningLogo': '\uED7C',
+      'Speakers': '\uE7F5',
+      'SpeedHigh': '\uEC4A',
+      'Split': '\uEDBC',
+      'Squalls': '\uE9CC',
+      'StackIndicator': '\uE7FF',
+      'Starburst': '\uEF78',
+      'StatusErrorFull': '\uEB90',
+      'StatusTriangle': '\uEA82',
+      'StockDown': '\uEB0F',
+      'StockUp': '\uEB11',
+      'Stopwatch': '\uE916',
+      'StoreLogo': '\uEA96',
+      'StoreLogoMed': '\uEA04',
+      'Strikethrough': '\uEDE0',
+      'Subscribe': '\uEDA1',
+      'Subscript': '\uEDDF',
+      'Suitcase': '\uEDD3',
+      'SunAdd': '\uEF69',
+      'SunQuestionMark': '\uEF6A',
+      'Sunny': '\uE9BD',
+      'Superscript': '\uEDDE',
+      'SwayLogo': '\uED29',
+      'Switcher': '\uE8AB',
+      'SwitcherStartEnd': '\uE810',
+      'Sync': '\uE895',
+      'SyncFolder': '\uE8F7',
+      'SyncToPc': '\uEE6E',
+      'System': '\uE770',
+      'Tab': '\uE7E9',
+      'Table': '\uED86',
+      'Tablet': '\uE70A',
+      'TabletSelected': '\uEC74',
+      'TaskManager': '\uEDB7',
+      'TaskManagerMirrored': '\uEDB8',
+      'Teamwork': '\uEA12',
+      'TemporaryUser': '\uEE58',
+      'Tennis': '\uEB33',
+      'TextBox': '\uEDC2',
+      'TextField': '\uEDC3',
+      'ThumbnailView': '\uE7B6',
+      'ThumbnailViewMirrored': '\uEA67',
+      'Thunderstorms': '\uE9C6',
+      'Ticket': '\uEB54',
+      'Tiles2': '\uEF7C',
+      'Timeline': '\uED9C',
+      'Timer': '\uE91E',
+      'ToggleBorder': '\uEC12',
+      'ToggleFilled': '\uEC11',
+      'ToggleThumb': '\uEC14',
+      'Touch': '\uE815',
+      'TouchPointer': '\uE7C9',
+      'Train': '\uE7C0',
+      'TrainSolid': '\uEB4D',
+      'TransferCall': '\uED95',
+      'Trash': '\uE74D',
```

Comment:
It wasn't generated by the subsetting tool.

TitleCased members was not the convention for enum collections, so when we moved the icons into styling, they got camelCased. 

`delete` is a keyword in javascript, so to avoid errors in the compiler/linting, we renamed it to `trash`.

In retrospect we should just keep them consistent and not try to be smart. So once Peter's tool can generate the "remainder" set, I'll update it. The `getIcon` helper is case insensitive so casing doesn't matter.